### PR TITLE
Km 5330 monitoring icon badge realignment

### PR DESCRIPTION
**Pull request by kiley-mitti**
## Brief Description

Retooled rux-monitoring-icon and rux-monitoring-progress-icon to:

1) Bring the designs more in line with figma (there had been some regressions on status, icon, and label placement) and

2) Reconfigure how the badge is placed.

If merged, this PR supersedes [PR 1028](https://github.com/RocketCommunicationsInc/astro/pull/1028) which can be closed and deleted.

Thanks to Micah's awesome change to deploy preview you can see some of my testing in the component deploy preview!

## JIRA Link

[ASTR0-5330](https://rocketcom.atlassian.net/browse/ASTRO-5330)

## General Notes
I pulled in the changes to rux-monitoring-progress-icon in this PR and then changed both of the components to be inline with each other.

Before merging I need to remove the content from /src/index.html

## Motivation and Context

The monitoring icon badge has been getting cut off by the GSB and any parent container that had `overflow: hidden` set. This reconfiguration ensures that the badge stays within the component boundaries so that that does not happen. It also 

## Issues and Limitations

May need to check with design to see if the way the badge expands now is ok. Figma has it a little differently but theirs expands outside the bounding box which will cause problems with overflow:hidden on a parent (like the GSB).

## Types of changes

- [x] Bug fix
- [ ] New feature
- [x] Breaking change - since it pretty drastically reconfigured the way rux-progress-monitoring-icon looks

## Checklist

- [ ] This PR adds or removes a Storybook story.
- [ ] I have added tests to cover my changes.
- [x] Regressions are passing and/or failures are documented
- [x] Changes have been checked in evergreen browsers

### CODE REVIEW

**Comment by micahjones13** [comment]
I think this might need a main merge, the `netlify.toml` still exists so the `astro-preview` is still just Storybook.

**Comment by kiley-mitti** [comment]
Absolutely right. Thanks Micah!!

**Comment by kiley-mitti** [comment]
@markacianfrani change sets added and I cleaned index.html I believe this is ready to merge when you are. Once we do we can close/delete [#1028](https://github.com/RocketCommunicationsInc/astro/pull/1028) The changes to Progress Icon are all in here.

**Comment by micahjones13** [comment]
Converted to draft and back to ready-for-review to trigger the tests, sorry if you got emailed

File: .changeset/fair-clouds-march.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
```
Comment (FMorrison87) [inline .changeset/fair-clouds-march.md:1]: Looks like there are three changesets. Can you delete two of them?

Comment (kiley-mitti) [reply .changeset/fair-clouds-march.md:1]: It actually needs all three for reference.. they're all related to different changes. :)

Comment (FMorrison87) [reply .changeset/fair-clouds-march.md:1]: Oh! Crazy.

**Review by FMorrison87 — APPROVED** [review]

**Comment by markacianfrani** [comment]
closing due to breaking change. backlog ticket created for v8

**Pull request closed without merging**